handlers/validator.go

```diff
@@ -121,7 +121,12 @@ func Validator(w http.ResponseWriter, r *http.Request) {
 		pubKey, err := hex.DecodeString(strings.Replace(vars["index"], "0x", "", -1))
 		if err != nil {
 			// logger.Errorf("error parsing validator public key %v: %v", vars["index"], err)
-			http.Error(w, "Internal server error", http.StatusInternalServerError)
+			SetPageDataTitle(data, "Validator not found")
```

Comment:
Please extract the "Validator not found" code in a separate function, to reduce code duplication. 

Including the "validatorNotFoundTemplate" and "validatorNotFoundTemplateFiles" variables.